pkg/ReadPackageJson.go

```diff
@@ -0,0 +1,46 @@
+package pkg
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+)
+
+type Scripts = map[string]string
+
+type IPackageJson struct {
+	Scripts Scripts `json:"scripts"`
+}
+
+func (p *IPackageJson) String() string {
+	var result []string
+	for name, command := range p.Scripts {
+		s := fmt.Sprintf("\x1b[32m%s\x1b[m:", name)
+		script := fmt.Sprintf("%-24s%s", s, command)
+		result = append(result, script)
+	}
+	return strings.Join(result, "\n")
+}
+
+func (p *IPackageJson) GetCommands() []string {
+	var commands []string
```

Comment:
長さが確定しているsliceはmakeで初期化すると多少最適化されるかも
```suggestion
	commands := make([]string, len(PackageJson.Scripts)
```

Reply:
なるほど